packages/polywrap-core/polywrap_core/utils/get_env_from_uri_history.py

```diff
@@ -1,10 +1,10 @@
-from typing import List, Union
+from typing import List, Union, Dict, Any
 
 from ..types import Client, Env, Uri
 
 
 def get_env_from_uri_history(
     uri_history: List[Uri], client: Client
-) -> Union[Env, None]:
+) -> Union[Dict[str, Any], None]:
```

Comment:
Why not use the `Env` type here? It helps me better understand what this code is doing.